lua/orgmode/org/mappings.lua

```diff
@@ -295,6 +295,20 @@ function OrgMappings:todo_prev_state()
   self:_todo_change_state('prev')
 end
 
+function OrgMappings:toggle_heading()
+  local node = Files.get_node_at_cursor()
```

Comment:
_nit_: Since we have the nearest section, we could compare line numbers rather than parsing nodes. Not entirely sure it makes any difference, but is perhaps more explicit.

```lua
if parent.line_number == vim.api.nvim_win_get_cursor(0)[1] then
  ...
```